### PR TITLE
Changes the vote sound

### PR DESCRIPTION
**Pull request by NullDagaf**
What it says in the title.
Did you know, that most people don't even notice the vote happening? That leads to people not voting, and in return vote results don't cover what the majority thinks.
This will make it so it's much less likely for you not to notice the vote.

### CODE REVIEW

File: code/controllers/subsystem/vote.dm [hunk 1]
```diff
@@ -242,7 +242,12 @@ SUBSYSTEM_DEF(vote)
 			V.Grant(C.mob)
 			generated_actions += V
 			if(C.prefs.toggles & SOUND_ANNOUNCEMENTS)
+			//SKYRAT EDIT START
+			/*
 				SEND_SOUND(C, sound('sound/misc/bloop.ogg'))
+			*/
+				SEND_SOUND(C, sound('sound/effects/adminhelp.ogg'))
```
Comment (Gandalf2k15) [inline code/controllers/subsystem/vote.dm]: This is gonna get a lot of complaints

**Comment by SpaceVampire** [comment]
Yeah. Don't make it the adminhelp sound.
I do my best to be civil in all github comments.
But so that is so fucking stupid, that if I am being polite.
I am going to assume you made this PR as a joke.

**Comment by NullDagaf** [comment]
> Yeah. Don't make it the adminhelp sound.
> I do my best to be civil in all github comments.
> But so that is so fucking stupid, that if I am being polite.
> I am going to assume you made this PR as a joke.

I kinda did. Point still stands though.
I'd change it to notice2.ogg , like it was on oldbase, buuut I have no idea how to make changes to PRs.

**Comment by GoldenAlpharex** [comment]
> > Yeah. Don't make it the adminhelp sound.
> > I do my best to be civil in all github comments.
> > But so that is so fucking stupid, that if I am being polite.
> > I am going to assume you made this PR as a joke.
> 
> I kinda did. Point still stands though.
> I'd change it to notice2.ogg , like it was on oldbase, buuut I have no idea how to make changes to PRs.

You just make a commit like you normally would before doing the PR.

**Comment by davidchan13** [comment]
This is kind of like having an air raid siren as your doorbell. Even if you think it's amusing, by the 5th time it goes off everyone else will be sick of it.

Goonstations 'Mapvote: Vote a Map' style of solution would be much better.

**Comment by NullDagaf** [comment]
> You just make a commit like you normally would before doing the PR.

Thanks.

File: code/controllers/subsystem/vote.dm [hunk 2]
```diff
@@ -242,7 +242,12 @@ SUBSYSTEM_DEF(vote)
 			V.Grant(C.mob)
 			generated_actions += V
 			if(C.prefs.toggles & SOUND_ANNOUNCEMENTS)
+			//SKYRAT EDIT START
+			/*
 				SEND_SOUND(C, sound('sound/misc/bloop.ogg'))
+			*/
+				SEND_SOUND(C, sound('sound/misc/notice2.ogg'))
```
Comment (Gandalf2k15) [inline code/controllers/subsystem/vote.dm]:
```suggestion
				SEND_SOUND(C, sound('sound/misc/announce_dig.ogg'))
```

**Comment by Gandalf2k15** [comment]
Change it to  'sound/misc/announce_dig.ogg' as my commit suggestion states, I think that'll have a much better effect on people.

**Pull request merged**